packages/core/build.mts

```diff
@@ -0,0 +1,21 @@
+import { build, defaultBuildOptions } from '@redwoodjs/framework-tools'
+
+// Build 'dist/bins'
```

Comment:
Nit-picky for sure, but aren't we actually building `src/bins` (and outputting to `dist/bins`)?

Reply:
It is just because you added an `import` that we now need to build? Can you use `require` instead, so we don't have to build?

Reply:
I'm happy to correct the comment.

Vitest doesn't natively intercept `require` which means if we still want to use `require` we'd need to look to some additional library. I think I'd rather build the files - keeps the source style consistent with the rest of the code and no additional dependency we're not already familiar with. See: https://github.com/vitest-dev/vitest/discussions/3134.

As we're getting increasingly close to removing webpack this test and so the need for vitest at all in this package will go away.